### ISSUE TITLE
Affected subsystem in commit message?

### ISSUE THREAD

**Issue by facekapow** [runtimejs/discussions#5]
I had an idea while commiting to the block subsystem on the `disk-driver` branch. How about adding the affected subsystem to the beginning of the commit message, the way node does? For example, `[block] Minor code style fixes`, or `[kernel] Move js loader into kernel, add onexit/onerror/promise handlers`, etc.

**Comment by iefserge** [comment]
Yeah good idea, more organized history is nice to have. Don't know how strict we should be with it though, since it's a bit of extra work to rename commits before merging a PR. I agree it would make it much easier to search commits.
